packages/suite/src/utils/wallet/trading/tradingUtils.ts

```diff
@@ -392,3 +392,22 @@ export const getTradeTypeByRoute = (
         return 'exchange';
     }
 };
+
+export const isUsdtToken = (contractAddress?: string) => {
+    const usdtContractAddresses = [
+        '0xdAC17F958D2ee523a2206206994597C13D831ec7', // ethereum
+        '0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9', // arbitrum one
+        '0x55d398326f99059fF775485246999027B3197955', // binance smart chain
+        '0x9702230a8ea53601f5cd2dc00fdbc13d4df4a8c7 ', // avalanche
+        '0xc2132D05D31c914a87C6611C10748AEb04B58e8F', // polygon
+        '0x94b008aa00579c1307b0ef2c499ad98a8ce58e58', // optimism
+    ];
```

Comment:
https://github.com/trezor/trezor-suite/issues/19220 this should not be hardcoded, the best scenario is to get some flag from trading backend